Podfile

```diff
@@ -4,19 +4,17 @@ source 'https://cdn.cocoapods.org/'
 
 # Definitions
 def shared_pods
-    # Version 1.5.3 has client-side symbolication turned off by default, which we want. Please don't upgrade to 1.5.4, which enables it by default.
-    pod 'Backtrace-PLCrashReporter', '1.5.3' 
+    # Define shared CocoaPods here
```

Comment:
in podfile definition shouldn't we have information about libraries that we used to have?

Reply:
Will populate it again for Cocoa & spm merge,
what do you suggest we put here in the meantime?